app/src/main/java/com/hedvig/app/feature/offer/OfferSignDialog.kt

```diff
@@ -119,11 +122,19 @@ class OfferSignDialog : DialogFragment() {
 
     private fun goToDirectDebit() {
         requireContext().storeBoolean(IS_VIEWING_OFFER, false)
+        val pref = PreferenceManager.getDefaultSharedPreferences(context)
+        val market = Market.values()[pref.getInt(Market.MARKET_SHARED_PREF, -1)]
```

Comment:
It's important that we don't default to `-1` as this will always cause a crash in case we're in an incorrect state.